index.md

```diff
@@ -61,24 +61,26 @@ debug issues across project boundaries.
 
 The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-1. commits MUST be prefixed with a type, which consists of a noun, `feat`, `fix`, etc.,
+1. Commits MUST be prefixed with a type, which consists of a noun, `feat`, `fix`, etc.,
    followed by a colon and a space.
-2. the type `feat` MUST be used when a commit adds a new feature to your application
+2. The type `feat` MUST be used when a commit adds a new feature to your application
   or library.
-3. the type `fix` MUST be used when a commit represents a bug fix for your application.
-4. an optional scope MAY be provided after a type. A scope is a phrase describing
+3. The type `fix` MUST be used when a commit represents a bug fix for your application.
+4. An optional scope MAY be provided after a type. A scope is a phrase describing
   a section of the codebase enclosed in parenthesis, e.g., `fix(parser):`
 5. A description MUST immediately follow the type/scope prefix.
   The description is a short description of the pull request, e.g.,
   _fix: array parsing issue when multiple spaces were contained in string._
-6. A longer commit body MAY be provided after the short description. The body MUST
-   begin one blank line after the description.
-7. A footer MAY be provided one blank line after the body. The footer SHOULD contain
-   additional meta-information about the pull-request (such as the issues it fixes, e.g., `fixes #13, #5`).
-8. Breaking changes MUST be indicated at the very beginning of the footer or body section of a commit. A breaking change MUST consist of the uppercase text `BREAKING CHANGE`, followed by a colon and a space.
-9. A description MUST be provided after the `BREAKING CHANGE: `, describing what
+6. A longer commit body MAY be provided after the short description.
+  The body MUST begin one blank line after the description.
+7. The body SHOULD only explain _what_ and _why_ the code changed.
+8. A footer MAY be provided one blank line after the body (or after the description if body is missing).
+  The footer SHOULD contain additional meta-information about the pull-request (such as the issues it fixes, e.g., `fixes #13, #5`).
+9. Breaking changes MUST be indicated at the very beginning of the footer or body section of a commit. A breaking change MUST consist of the uppercase text `BREAKING CHANGE`, followed by a colon and a space.
+10. A description MUST be provided after the `BREAKING CHANGE: `, describing what
   has changed about the API, e.g., _BREAKING CHANGE: environment variables now take precedence over config files._
-10. types other than `feat` and `fix` MAY be used in your commit messages.
+11. The footer SHOULD only contain `BREAKING CHANGE`, external links or meta-information.
```

Comment:
if i just look at this, I'm not sure what meta-information actually means... Maybe we should call it issue references?